README.md

```diff
@@ -635,3 +635,41 @@ the following in this order:
 ```
 Line `N` in the report should correspond to line `N` in the batch file, and you
 can also use the batch-id for correlation.
+
+#### Ansible Vault support
+
+If you want to test using variables encrypted with
+[Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) do the
+following:
+* put the password in the file `tests/vault_pwd` e.g. `echo -n password > tests/vault_pwd`
+* `mkdir tests/vars`
+* encrypt the variables
+```
+ansible-vault encrypt_string --vault-password-file tests/vault_pwd my_secret_value \
+  --name my_secret_var_name >> tests/vars/vault-variables.txt
```

Comment:
Could this file's extention be `.yml`? `runqemu.py` handles `vault-variables.yml` and `no-vault-variables.txt`.